torch/testing/_internal/common_device_type.py

```diff
@@ -1083,6 +1096,9 @@ def _has_sufficient_memory(device, size):
             device = 'cuda:0'
         return torch.cuda.memory.mem_get_info(device)[0] >= size
 
+    if device == 'xpu':
+        raise unittest.SkipTest('TODO: Memory availability checks for XPU?')
+    
```

Comment:
Why do we need such a check here? We need to follow cuda.

Reply:
We still has feature gap to support memory check, shall we enable memory check and remove these two lines later?